### DISCUSSION TITLE
Can the lights be controlled?

### DISCUSSION THREAD

**Discussion by bangcorrupt** [bangcorrupt/hacktribe#214] [Q&A] [answered]
lenlux asks:

The BRIGHTNESS of the display as well as the built-in LEDs - especially those 4 blinking lights on the base-plate. I see there is a powersaving mode which turns off these 4 LEDS on the bottom and also reduces the display brightness - but then there is also very bad PWM-like flickering which actually makes looking at the screen unbearable.
Do you reckon one could add an option to Hacktribe to turn down the display brightness as well as turn off/down the other LEDs on the unit? (Especially when performing in the dark or low light this would really help I think. For now I put (layers of tape) over the display and LEDs)

**Comment by bangcorrupt** [comment, marked as answer]
Controlling the light show should be possible at some point.  It would be cool if we could program it from the sequencer.  I think the only way to dim the screen will be PWM as the light will be controlled by a digital output.  There is a way to change the screen colour somehow, but I haven't found it in the firmware yet.

#109

**Reply by lenlowe** [reply]
Thank you for answering! For now I get by with enough tape, the occasional squinting in the dark and the option of sunglasses :>

I think turning off the light show would be the easiest task since the existing powersaving-mode already does that. So that would probably be most easily achievable and also potentially beneficial for powersaving. Am I correct in assuming adding custom configuration menu options/categories to the firmware is a problem at this point? In this case maybe could such a option (e.g. turning off the lightshow) be "hardcoded" into the patched firmware?

I'm afraid you're probably right about PWM being the only way to dim down the LCD screen brightness - if there was a better option the Korg devs would likely have used it instead of this flickering mess in powersaving-mode as is. Yet maybe changing the default color to one of the other available modes (red/blue/..?) as in the other discussion you linked might actually help a bit. The blue screen especially seems quite a lot less blinding than the red and green on first sight.

For this as well: Maybe offering a configuration file to edit before patching the firmware file would be a good option rather than trying to add new menu options to the firmware if that is indeed problematic. I for one would be quite content with finding a preferred setting and sticking with it so there would be no need for custom menus for on-the-fly changing of settings.

Thanks again and all the best!

**Comment by BKLronin** [comment]
Random fact I learned recently: 

The lights are actually the VU meter of master. Red being in the clipping range. So its kinda helpful too.
